src/assets/images/home/index.js

```diff
@@ -6,6 +6,7 @@ import styled from 'styled-components';
 
 import DataSecurityAndCompliancePNG from './data-security-and-compliance.png';
 import LaptopPNG from './laptop.png';
+import PhonePNG from './phone.png';
```

Comment:
has this been compressed already? .3MB seems pretty heavy. https://tinypng.com/ or https://squoosh.app/ are great online tools to do this very quickly

Reply:
i actually have compressed it once already... was originally 1.8MB